### DISCUSSION TITLE
the hold for menu button is distracting as all hell

### DISCUSSION THREAD

**Discussion by DeadEyed-5** [ppy/osu#17308] [Q&A] [answered]
i cant play a single beatmap without having the hold for menu X pop up and its the most distracting thing in the world. is there a way to disable the auto pop up whenever your cursor goes near it. 
edit: my screen is monitor is square because its all i have so the play area takes up the whole monitor basically. So every time i go near the bottom right it the hold for menu button shows up.

**Comment by frenzibyte** [comment, marked as answer]
Escalated to #17313
